authors/apps/articles/models.py

```diff
@@ -1,8 +1,39 @@
 from django.db import models
 from .utils import generate_slug, unique_code_generator, get_likes_or_dislkes
 from django.db.models.signals import pre_save
+from authors.apps.authentication.models import User 
 from authors.apps.profiles.models import Profile
-from authors.apps.authentication.models import User
+from authors.apps.utils.messages import error_messages, favorite_actions_messages
+
+
+class ArticleManager(models.Manager):
+    def handle_favorite_actions(self, request_user_obj, article_slug):
+        """ 
+        Enales both favoriting and unfavoriting of articles 
+        Args:
+            request_obj: 
```

Comment:
@armstrongsouljah Please fully define all arguments

Reply:
okay. Let me work on that.